advanced/sr_gui_motor_resetter/src/sr_gui_motor_resetter/motor_resetter.py

```diff
@@ -185,6 +186,7 @@ def populate_motors(self):
             row += 1
 
     def diagnostics_callback(self, msg):
+        # pylint: disable=R1702
         for status in msg.status:
```

Comment:
Is there no way to simplify this?

Reply:
I could not see a way of doing it, everything here felt necessary